benchmarks/src/main/scala/akka.kafka.benchmarks/KafkaConsumerBenchmarks.scala

```diff
@@ -71,6 +71,8 @@ object KafkaConsumerBenchmarks extends LazyLogging {
 
     var lastProcessedOffset = 0L
     var accumulatedMsgCount = 0L
+    // TODO this is actually running single threaded, commitAsync is called on the poll thread.
+    //      It might not be waiting as it should for the commits to be completed?
     val semaphore = new Semaphore(1)
```

Comment:
The other issue is that this test keeps polling while the commit is in flight and hence it does not measure the same thing as the reactive connector. The Akka consumer test will actually not request form the upstream in mapAsync until the commit has been finished, which means that topics are paused in the consumer actor until the commit finishes. The two are not really comparable.